thelper/nn/utils.py

```diff
@@ -182,6 +182,12 @@ def create_model(config, task, save_dir=None, ckptdata=None):
         model.config = model_params
     if hasattr(model, "summary"):
         model.summary()
+    param_count = get_learnable_param_count(model)
+    if param_count > 10 ** 5:
+        param_count_str = f"{(param_count // 10 ** 5) / 10}M"
+    else:
+        param_count_str = f"{param_count}"
```

Comment:
when in range 100K>params>1M, this looks a bit weird, and most precision is lost
eg: 
```
f"{(125000 // 10 ** 5) / 10}M"
'0.1M'
```

Reply:
ahhh the good old days of small models are far...